modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -460,24 +453,29 @@ class ApiClient(object):
     def files_parameters(self, files=None):
         """Builds form parameters.
 
-        :param files: File parameters.
-        :return: Form parameters with files.
+        :param files: None or a dict with key=param_name and value=file_object
+        :return: List of tuples of form parameters with file data
         """
-        params = []
+        if files is None:
+            return []
 
-        if files:
-            for k, v in six.iteritems(files):
```

Comment:
so it looks like this was storing:
```
{
    'file_param_a': 'file_path_1',
    'file_param_b': [
        'file_path_1',
        'file_path_2',
    ]
}
```
I am unsure if my update handles this correctly. Want me to add a spec endpoint + test for this?

Reply:
Sure. I think python-experimental is using its own spec for testing as part of this PR: modules\openapi-generator\src\test\resources\2_0\python-client-experimental\petstore-with-fake-endpoints-models-for-testing.yaml

Reply:
I added a test which verifies that file uploading is working.
The updated code stores and handles:
```
{
    'file_param_a': ['file_path_1'],
    'file_param_b': [
        'file_path_1',
        'file_path_2',
    ]
}
```